onnxruntime/test/python/onnxruntime_test_cudagraph.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# -*- coding: UTF-8 -*-
+import unittest
+import os
+import numpy as np
+import gc
+
+import onnxruntime as onnxrt
+import threading
+import sys
+from helper import get_name
+from onnxruntime.capi.onnxruntime_pybind11_state import Fail
+import time
+
+class TestInferenceSessionWithCudaGraph(unittest.TestCase):
```

Comment:
This script should be run as part of the build process in CI. Please hook it up to build.py. See this for an example- https://github.com/microsoft/onnxruntime/blob/baa17679221807ea332a7134c0d10b81adf4009e/tools/ci_build/build.py#L1614